ee/agent/reset.go

```diff
@@ -340,16 +340,13 @@ func getLocalPubKey(k types.Knapsack) ([]byte, error) { // nolint:unused
 	return pubKeyBytes, nil
 }
 
-// wipeDatabase iterates over all stores in the database, deleting all keys from
+// WipeDatabase iterates over all stores in the database, deleting all keys from
 // each one.
-func wipeDatabase(ctx context.Context, k types.Knapsack) error { // nolint:unused
-	return errors.New("currently not supported")
-	/*
-		for storeName, store := range k.Stores() {
```

Comment:
Why was this commented out? Was it an additional safety for `DetectAndRemediateHardwareChange`, which looks like it's not calling anyhow? (@RebeccaMahany ?)